### ISSUE TITLE
[1.16.4] Json file for the ores are missing the block properties

### ISSUE THREAD

**Issue by TelepathicGrunt** [klikli-dev/occultism#91]
Hello! Someone was running my mod called Blame with Occultism and found some errors came up about Occultism's ore blocks. 
https://github.com/TelepathicGrunt/Blame/issues/20

Here's an example entry of what Blame spit out in the logs:
```
****************** Blame Report 1.8.1 ******************

 Failed to load resource file: occultism:worldgen/configured_feature/otherworld_tree_natural.json

 Reason stated: No key Properties in MapLike[{"Name":"occultism:iesnium_ore_natural"}]

 Possibly helpful hint (hopefully): If this is a worldgen JSON file, check out slicedlime's example datapack
   for worldgen to find what's off about the JSON: https://t.co/cm3pJcAHcy?amp=1

 Prettified JSON: 
 Unable to display JSON. 


[12Jan2021 14:49:08.930] [Render thread/ERROR] [com.telepathicgrunt.blame.Blame/]: Failed to turn error msg into string. Please notify TelepathicGrunt (Blame creator) and show him this message:  No key Properties in MapLike[{"Name":"occultism:iesnium_ore_natural"}]
```

What this is saying is that the JSON files is missing the properties entry for the block to use. First I checked out the iesnium_ore in code and it seems it has a property called "UNCOVERED"
https://github.com/klikli-dev/occultism/blob/d57e79298c0c4a5d60d63d4152043a8cb2350891/src/main/java/com/github/klikli_dev/occultism/common/block/otherworld/IesniumOreNaturalBlock.java#L45

Then I checked the JSON and it only says the block itself but not what property to use which is why the codec parsing exploded for this configuredfeature. If your ore was not generating in the world, this would be why I bet.
https://github.com/klikli-dev/occultism/blob/d57e79298c0c4a5d60d63d4152043a8cb2350891/src/main/resources/data/occultism/worldgen/configured_feature/ore_iesnium.json#L7-L9

Basically, state should look like this to be valid JSON for Minecraft to parse for all of your blocks with properties. If there's more than one property, just add another line in the Properties entry.
```
       "state": {
          "Properties": {
             "uncovered": "false"
           },
           "Name": "occultism:iesnium_ore"
        },
```

Hope this helps!

**Comment by klikli-dev** [comment]
Thanks a lot, I will update this!
Interestingly, as far as I can tell, the ore does generate, I assume it might fall back to default state? 
Anyway, best to fix it, so thanks again!
